packages/tools/devtools/devtools-core/src/CommonInterfaces.ts

```diff
@@ -56,4 +56,6 @@ export enum EditType {
 	Number = "number",
 	String = "string",
 	Boolean = "boolean",
+	Null = "null",
```

Comment:
You're going to have a minor merge conflict here. I can help resolve if you have any questions.